packages/terra-worklist-data-grid/src/subcomponents/ColumnHeaderCell.module.scss

```diff
@@ -38,8 +38,23 @@
 
   // Hover state for selectable elements
   .selectable:hover {
-    background-color: var(--terra-worklist-data-grid-hover-background-color, rgba(0, 146, 224, 0.12));
+    background-color: var(--terra-worklist-data-grid-header-hover-background-color, #e4f2f9);
     cursor: pointer;
   }
-}
 
+  .pinned {
+    border: inherit, 1px solid #fff;
```

Comment:
Don't you need to account for theming with your colors?  With this implementation, there is no option to change the color based on the theme.  Same for other newly introduced colors.

Reply:
You are correct that we need to account for theming; we will work with UX after this PR is merged to incorporate theming.

See Eric's comment: https://github.com/cerner/terra-framework/pull/1710#issuecomment-1650586532

CC: @eawww

Reply:
Even if you work with UX later, shouldn't we establish the proper variables in this PR?

Reply:
Added variables here: b358091d9295942c9a2bcc5ce4e02f8aa9113773